server/tokenapi/Main.go

```diff
@@ -0,0 +1,50 @@
+package tokenapi
+
+import (
+	"github.com/aerogo/aero"
+	"github.com/aerogo/log"
+	"github.com/akyoto/uuid"
+	"github.com/animenotifier/notify.moe/arn"
+	"github.com/tidwall/gjson"
+
+	"net/http"
+)
+
+type TokenRequest struct {
+	Token uuid.UUID
+	User  arn.User
+
+	JSON gjson.Result
+}
+
+func Main(app *aero.Application, log *log.Log) {
+	app.Post("/tokenapi", func(ctx aero.Context) error {
+		response, err := ctx.Request().Body().String()
+		if err != nil {
```

Comment:
I can't make everyone agree on styling conventions, but it's better to have them than to have none.

See [empty line styling conventions](https://github.com/akyoto/quality/blob/master/STYLE.md#empty-line-between-blocks-and-statements).